NEWS.md

```diff
@@ -14,7 +14,8 @@
 * Identify non-riskmetric cards (#573)
 
 ### Squashed Bugs
-* Fixed busted button introduced with #547 (#592)
+* Fixed busted button introduced with #547 
```

Comment:
I think we should keep the habit of tracking PRs in the news file. Could you leave the (# 592)?
```suggestion
* Fixed busted button introduced with #547 (#592)
```